lib/filepicker_rails/filepicker_image_url.rb

```diff
@@ -0,0 +1,141 @@
+# Creates the full path of the image to the specified `url` accepts optional
+# `options` hash for configuration.
+#
+# #### Constructor
+#
+# @param [String] url                       Base filepicker url
+# @param [Hash] options
```

Comment:
:+1: on removing options with a reference to the helper

My bad on the failing docs. I hadn't tried to compile them. I'll give it a whirl and fix it up.